### PR TITLE
Confinement symbology for Q

### PR DESCRIPTION
**Pull request by lauren-herbine**
Everything but flowlines are uniquely symbolized for a confinement project.

### CODE REVIEW

File: RaveBusinessLogic/Confinement.xml [hunk 1]
```diff
@@ -12,21 +12,21 @@
           <Node xpathlabel="Name" xpath="Outputs/Geopackage/Layers/Vector[@id='CONFINEMENT_MARGINS']" type="line" id="confinement_margins" symbology="confinement_margins" />
           <Node xpathlabel="Name" xpath="Outputs/Geopackage/Layers/Vector[@id='CONFINEMENT_RATIO']" type="line" id="confinement_ratio" symbology="confinement_ratio" />
           <Node label="Constriction Ratio" xpath="Outputs/Geopackage/Layers/Vector[@id='CONFINEMENT_RATIO']" type="line" id="constriction_ratio" symbology="constriction_ratio" />
```
Comment (joewheaton) [inline RaveBusinessLogic/Confinement.xml:14]: So is this actually a "Constriction Ratio"? The xpath says "Confinement Ratio"? Two problems here.
- [ ] Make sure this is actually coming from the "Constriction_Ratio" field in attribute table (we know it is from [here](https://github.com/Riverscapes/RiverscapesXML/blob/confinement_project/Symbology/qgis/Confinement/constriction_ratio.qml#L15).
- [ ] Label and Display as a 0 - 1 value (not a binary) as "Constriction Proportion". 

Constriction Ratio in the literature often refers to a discrete constriction point (e.g. a debris fan rapid) and is a ratio of upstream or downstream channel width to width at constriction. We proposed in paper Constriction proportion as something analogous to the confinement ratio, in that it is the percentage of that reach length that is confined on both sides. See [this issue](https://github.com/Riverscapes/riverscapes-tools/issues/316)

File: Symbology/qgis/Confinement/constriction_ratio.qml [hunk 2]
```diff
@@ -0,0 +1,479 @@
...
+  <temporal enabled="0" durationUnit="min" endExpression="" durationField="" startField="" endField="" mode="0" accumulate="0" startExpression="" fixedDuration="0">
+    <fixedRange>
+      <start></start>
+      <end></end>
+    </fixedRange>
+  </temporal>
+  <renderer-v2 type="graduatedSymbol" forceraster="0" symbollevels="0" graduatedMethod="GraduatedColor" enableorderby="0" attr="Constriction_Ratio">
+    <ranges>
+      <range render="true" upper="0.000000000000000" lower="0.000000000000000" label="Not constricted" symbol="0"/>
+      <range render="true" upper="1.000000000000000" lower="0.000000000000000" label="Constricted" symbol="1"/>
```
Comment (joewheaton) [inline Symbology/qgis/Confinement/constriction_ratio.qml:18]: This should be something similar to confinement ratio as in NOT binary.  There is value in having constrictions highlighted discretely, but those are generally smaller than the total reach length and come from the Confinement Raw stuff (i.e. both).

**Comment by MattReimer** [comment]
@lauren-herbine what's the status here. Is this ready for review?

**Comment by lauren-herbine** [comment]
Hi @MattReimer, no, not yet- sorry I have recently refocused on some other parts of the RS project

**Review by joewheaton — APPROVED** [review]
I'm approving for now, but I didn't see where the screen shots were? I think that might be because this was before we had that template. Anyhow, lets run with this for now and see how we like it.

**Comment by lauren-herbine** [comment]
@joewheaton and @MattReimer I'm going to take a quick look at this again (it's been a couple of months)... but if I like what I see I'll add some documentation here

**Comment by lauren-herbine** [comment]
@MattReimer this should be good to go now

**Pull request merged**